### PR TITLE
make deltachat-node build on mac

### PR DESCRIPTION
**Pull request by Jikstra**
Currently this needs to have deltachat-core folder replaced with the fix_for_mac branch to work (i changed some stuff and used git submodule magic, but not sure if it works)

### CODE REVIEW

File: binding.gyp [hunk 1]
```diff
@@ -18,7 +18,7 @@
         [ "OS == 'mac'", {
           "libraries": [
             "../deltachat-core/builddir/src/libdeltachat.a",
-            "../deltachat-core/builddir/libs/libetpan/libetpan.a",
+            "/usr/local/Cellar/libetpan/1.9.2_1/lib/libetpan.a",
```
Comment (ralphtheninja) [inline binding.gyp]: I hope this will be the same on each machine. We might want/need to parametrize it somehow later.

Comment (Jikstra) [reply binding.gyp]: We should also be able to use `/usr/local/lib/libetpan.a` will give it a try

**Comment by ralphtheninja** [comment]
Wtf, something is borked with Jenkins as well. Ignore Jenkins for now.

File: binding.gyp (hunk 1 above)
Comment (Jikstra) [reply binding.gyp]: But maybe it's even more clean to use this `libetpan-config` tool.

Comment (ralphtheninja) [reply binding.gyp]: Maybe yeah, but lets get it to work first :wink:

**Pull request merged**